Examples/Tests/ohm_solver_magnetic_reconnection/inputs_test_2d_ohm_solver_magnetic_reconnection_picmi.py

```diff
@@ -306,15 +322,25 @@ def check_fields(self):
         if not (step == 1 or step % self.diag_steps == 0):
             return
 
-        rho = simulation.fields.get("rho_fp", level=0)[...]
-        Jiy = simulation.fields.get("current_fp", dir="y", level=0)[...] / self.J0
-        Jy = (
+        get_xp()
+
+        rho = to_host(simulation.fields.get("rho_fp", level=0)[...])
+
+        Jiy = to_host(simulation.fields.get("current_fp", dir="y", level=0)[...])
```

Comment:
Do this instead and stay on device:
```suggestion
        rho = simulation.fields.get("rho_fp", level=0)

        Jiy = simulation.fields.get("current_fp", dir="y", level=0).mult(1 / self.J0)
```
https://pyamrex.readthedocs.io/en/25.11/usage/api.html#amrex.space3d.MultiFab.mult